lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -280,8 +280,14 @@ def add_job_logs(self, job_id, log_lines):
             ts = input_line.get("ts")
             # TODO Maybe use strpos for efficiency?
             if ts is not None:
-                if type(ts) == str:
-                    ts = dateutil.parser.parse(ts)
+                try:     # input ts as float/int/str epoch timestamp
+                    ts = float(ts)
+                except Exception:
+                    try:  # input ts as datetime string
+                        ts = dateutil.parser.parse(ts).timestamp()
+                    except Exception:
+                        ts = None  # TODO raise error?
```

Comment:
You could set TS to now instead of setting it to None

Reply:
Not sure if that would cause problems though with log ordering.... But I think the ordering is done on the position, not the timestamp..

Reply:
👍 I think `LogLines` will default ts to utcnow() if `None`. But it doesn't hurt.